src/rp/util/time.clj

```diff
@@ -10,7 +10,9 @@
 (def patterns
   {:review      "dd-MMM-yyyy"
    :floorplan   "MM/dd/yyyy"
-   :last-update "MM/dd/yyyy HH:mm:ss"})
+   :last-update "MM/dd/yyyy HH:mm:ss"
+   ;; Used for terminate_dm property in Rent Endeca
+   :terminate   "yyyyMMddHHmmss.SSSSSS"})
```

Comment:
I don't like that this util repo identifies formats by their use in Endeca, but since this pattern is already in place I suppose I'll go along with it for now. Perhaps we might consider cleaning it up in the future...